hybridse/include/codec/fe_row_codec.h

```diff
@@ -217,6 +217,89 @@ class RowFormat {
     uint32_t str_field_start_offset_;
 };
 
+class RowFormat {
+ public:
+    virtual bool GetStringColumnInfo(size_t schema_idx, size_t idx, StringColInfo* res) const = 0;
+    virtual const ColInfo* GetColumnInfo(size_t schema_idx, size_t idx) const = 0;
+    virtual size_t GetSliceId(size_t schema_idx) const = 0;
+};
+
+class MultiSlicesRowFormat : public RowFormat {
```

Comment:
what's the different between `SliceFormat` and `RowFormat`. And is has some confused that the relationship between `MultiSlicesRowFormat` and `SliceFormat`